db/migrate/20191209095548_ensures_unique_accepted_answer_post_id.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class EnsuresUniqueAcceptedAnswerPostId < ActiveRecord::Migration[6.0]
```

Comment:
For stable branch compatability, this should be
```suggestion
class EnsuresUniqueAcceptedAnswerPostId < ActiveRecord::Migration[5.2]
```